device/hugepage.cpp

```diff
@@ -8,15 +8,13 @@
 
 #include <fcntl.h>     // for O_RDWR and other constants
 #include <sys/stat.h>  // for umask
+#include <unistd.h>    // for unlink
 
-#include "cpuset_lib.hpp"
-#include "logger.hpp"
+#include <regex>
 
-const uint32_t g_MAX_HOST_MEM_CHANNELS = 4;
+#include "logger.hpp"
 
-// Hardcode (but allow override) of path now, to support environments with other 1GB hugepage mounts not for runtime.
-const char* hugepage_dir_env = std::getenv("TT_BACKEND_HUGEPAGE_DIR");
```

Comment:
I tried deleting this at some point as well and @broskoTT had useful comment that maybe some UMD client relies on this env var. From TT_BACKEND prefix I assume that it might be budabackend so it should not be the problem. I created an issue to investigate usage of env vars #146, so if you end up deleting this just write down there that this one is deleted

Reply:
I'll update the issue after this gets merged.